src/plugins/opensearch_dashboards_legacy/public/index.ts

```diff
@@ -38,4 +38,4 @@ export * from './plugin';
 
 export { PaginateDirectiveProvider, PaginateControlsDirectiveProvider } from './paginate/paginate';
 export * from './notify';
-export * from './utils';
+// export * from './utils';
```

Comment:
If you are deleing all the exports from `utils`, feel free to delete this commented line as well